rotkehlchen/db/history_events.py

```diff
@@ -108,6 +126,12 @@ def edit_history_event(self, event: HistoryBaseEntry) -> tuple[bool, str]:
                 )
                 return False, msg
 
+            if isinstance(event, EvmEvent):
+                cursor.execute(
+                    'UPDATE evm_events_info SET protocol=?, product=?, address=? WHERE identifier=?',    # noqa: E501
+                    (*event.serialize_evm_event_for_db_without_extra_data(), event.identifier),
+                )
```

Comment:
shouldn't this catch integrity error as we do when updating `history_events` ?

Reply:
No since there are no constraints on the values (like unique, primary key, etc.) touched by this statement.